distributed/scheduler.py

```diff
@@ -655,6 +655,12 @@ def _task_key_or_none(task):
     return task.key if task is not None else None
 
 
+if sc.__build__ < 0x020000:  # for backwards compatibility
```

Comment:
I'm curious about the definition of `__build__` here.  Would it make sense to check something like `sc.__version__ < '2.0.0'`, which seems more common?

Reply:
It would be more readable but strings are flawed for these kinds of comparisons. I wouldn't want version "11.0.0" to be less than "2.0.0".

The `__build__` attribute makes the same guarantees as [`sys.hexversion`](https://docs.python.org/3.6/library/sys.html#sys.hexversion) in the standard library: 1) version encoded as a single integer; 2) guaranteed to increase with each version.

For prior art on naming, see the `requests` module: https://github.com/requests/requests/blob/master/requests/__version__.py#L9

Reply:
How about using `LooseVersion`?

```Python
from distutils.version import LooseVersion
LooseVersion(sc.__version__) < LooseVersion("2.0.0"):
```
https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Reply:
`LooseVersion` is a good idea. Thanks for the tip! I updated the code.